content/en/docs/tasks/observability/metrics/customize-metrics/index.md

```diff
@@ -174,21 +174,23 @@ has(request.host) ? request.host : "unknown"
 
 For more information, see [Common Expression Language](https://opensource.google/projects/cel).
 
-Istio exposes all standard [Envoy attributes](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/security/rbac_filter#condition).
-Additionally, you can use the following extra attributes.
+Istio exposes all standard [Envoy attributes](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/advanced/attributes).
+Peer metadata is available as attributes `upstream_peer` for outbound and `downstream_peer` for inbound with the following fields:
 
-|Attribute   | Type  | Value |
+|Field   | Type  | Value |
 |---|---|---|
-| `listener_direction` | int64 | Enumeration value for [listener direction](https://www.envoyproxy.io/docs/envoy/latest/api-v2/api/v2/core/base.proto#envoy-api-enum-core-trafficdirection) |
-| `listener_metadata` | [metadata](https://www.envoyproxy.io/docs/envoy/latest/api-v2/api/v2/core/base.proto#core-metadata) | Per-listener metadata |
-| `route_metadata` | [metadata](https://www.envoyproxy.io/docs/envoy/latest/api-v2/api/v2/core/base.proto#core-metadata) | Per-route metadata |
-| `cluster_metadata` | [metadata](https://www.envoyproxy.io/docs/envoy/latest/api-v2/api/v2/core/base.proto#core-metadata) | Per-cluster metadata |
-| `node` | [node](https://www.envoyproxy.io/docs/envoy/latest/api-v2/api/v2/core/base.proto#core-node) | Node description |
-| `cluster_name` | string | Upstream cluster name |
-| `route_name` | string | Route name |
-| `filter_state` | map[string, bytes] | Per-filter state blob |
```

Comment:
do we need to provide guidance around how to access things like `filter_state` moving forward ?

Reply:
`filter_state` is documented in the Envoy link. There's a lack of Envoy Wasm docs, but once they are written, they should also cover filter state for Wasm specifically.